homework.md

```diff
@@ -44,37 +44,40 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+An object-oriented computer programming language commonly used to create interactive effects within web browsers.
```

Comment:
You see on your answer if you had ended  on:
> An object-oriented computer programming language
>
Now that would have been enough because they asked, "What is JavaScript" not "What does JavaScript do".

# Plus
- When you add "What JavaScript does" you have to take into consideration that JavaScript is more than just for making interactive effects in web browsers, you can build **CLI's, Web Services, Mobile Applications and Desktop Application etc...**